deployer/README.md

```diff
@@ -121,6 +121,7 @@ are available:
 * `es-pvc-size`: Size of the PersistentVolumeClaim to create per ElasticSearch ops instance, e.g. 100G. If empty, no PVCs will be created and emptyDir volumes are used instead.
 * `es-pvc-prefix`: Prefix for the names of PersistentVolumeClaims to be created; a number will be appended per instance. If they don't already exist, they will be created with size `es-pvc-size`.
 * `es-pvc-dynamic`: Set to `true` to have created PersistentVolumeClaims annotated such that their backing storage can be dynamically provisioned (if that is available for your cluster).
+* `storage-group`: Number of a supplemental group ID for access to Elasticsearch storage volumes; backing volumes should allow access by this group ID (defaults to 1095).
```

Comment:
Thanks, missed one

On Wed, Jun 22, 2016 at 7:17 PM, Richard Megginson <notifications@github.com
> wrote:

> In deployer/README.md
> <https://github.com/openshift/origin-aggregated-logging/pull/181#discussion_r68152474>
> :
>
> > @@ -121,6 +121,7 @@ are available:
> >  * `es-pvc-size`: Size of the PersistentVolumeClaim to create per ElasticSearch ops instance, e.g. 100G. If empty, no PVCs will be created and emptyDir volumes are used instead.
> >  * `es-pvc-prefix`: Prefix for the names of PersistentVolumeClaims to be created; a number will be appended per instance. If they don't already exist, they will be created with size `es-pvc-size`.
> >  * `es-pvc-dynamic`: Set to `true` to have created PersistentVolumeClaims annotated such that their backing storage can be dynamically provisioned (if that is available for your cluster).
> > +* `storage-group`: Number of a supplemental group ID for access to Elasticsearch storage volumes; backing volumes should allow access by this group ID (defaults to 1095).
>
> This says 1095 but below you use 65534
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/181/files/83de46be43764964be7ded8481815cef54adac6a#r68152474>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABz-mPEBE66kfd6edEIyy0sAjYZpH_5ks5qOcJ4gaJpZM4I8MEq>
> .
>